auto-scale.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_appautoscaling_target" "aurora_scaling_target" {
+  count              = var.enable_auto_scale ? 1 : 0
+  service_namespace  = "rds"
+  resource_id        = "cluster:${aws_rds_cluster.primary.id}"
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
+  min_capacity       = var.auto_scale_min_capacity
+  max_capacity       = var.auto_scale_max_capacity
+}
+
+resource "aws_appautoscaling_policy" "aurora_scaling_policy" {
+  count              = var.enable_auto_scale ? 1 : 0
+  name               = "${var.auto_scale_metric} policy"
+  service_namespace  = "rds"
+  resource_id        = aws_appautoscaling_target.aurora_scaling_target[0].resource_id
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
```

Comment:
Same comment as above.

Reply:
Changed to reference the value defined on the `aws_appautoscaling_target` resource